Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -2898,6 +2902,109 @@ bool Element::has_paint_containment() const
     return false;
 }
 
+int Element::number_of_owned_list_items() const
```

Comment:
use types from AK/Types.h over int in almost every case for new code.

For this, size_t might be suitable instead of an explicitly-sized type like `u32` or `u64`.